v2/handler.js

```diff
@@ -705,12 +708,8 @@ function validateCallResponseFrame(res) {
         assert(res.headers && res.headers.as,
             'Expected the "as" transport header to be set for response');
     } else if (!res.headers || !res.headers.as) {
-        this.logger.error('Expected "as" header to be set for response', {
-            code: res.code,
-            remoteName: this.remoteName,
-            arg1: this.inreq.endpoint,
```

Comment:
btw this was a subtle uncaught exception waiting to happen in a rare error handling path